### DISCUSSION TITLE
Local VST integration

### DISCUSSION THREAD

**Discussion by djniviro** [apohl79/audiogridder#889] [Ideas]
As a touring artist, I was wondering if there is a way to be able to work in my projects using the AudioGridder VST on the road. Basically as how the Waves Studiorack (SoundGrid network) works. This could be a life-saver for most of us. Of course this will need the criteria that all VSTs used on the AG server must be installed on the local machine using the same plugin version.

Is there a way to make the AG VST plugin scan whether a server is connected, and if not, it loads the VST locally with its settings.

Even more advanced would be a selection button to choose whether to run the plugin locally or on the server for maximum flexibility.

Keep me posted!

**Comment by CaBuffalo** [comment]
For now, AG scans the servier and then let you choose which one to connect
with. Doesn't this work for you?

On Thu, Jan 20, 2022 at 1:06 PM djniviro ***@***.***> wrote:

> As a touring artist, I was wondering if there is a way to be able to work
> in my projects using the AudioGridder VST on the road. Basically as how the
> Waves Studiorack (SoundGrid network) works. This could be a life-saver for
> most of us. Of course this will need the criteria that all VSTs used on the
> AG server must be installed on the local machine using the same plugin
> version.
>
> Is there a way to make the AG VST plugin scan whether a server is
> connected, and if not, it loads the VST locally with its settings.
>
> Even more advanced would be a selection button to choose whether to run
> the plugin locally or on the server for maximum flexibility.
>
> Keep me posted!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/889>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6G2ULRD3YOOOKV4TI3UXB2MXANCNFSM5MN2A4DQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by djniviro** [reply]
It might be that I missed something, but the primary question is: is there a way to be able to run the project with the AG VST without having a server connected (thus loading in the plugins on the AG VST and not on the AG SERVER)?

**Reply by emack999** [reply]
If you have the plugins also installed on the client machine, you can get what you're looking for by running AG Server on that machine.  You'll have to open each instance of the AG plugin and point it toward the local server you just ran(rather than the remote server it was used to looking for), but it should work.  There's a Local Mode option in the AG Server settings.

**Comment by CaBuffalo** [comment]
Exactly, you need the AG server regardless.

On Thu, Jan 20, 2022 at 5:36 PM emack999 ***@***.***> wrote:

> If you have the plugins also installed on the client machine, you can get
> what you're looking for by running AG Server on that machine. You'll have
> to open each instance of the AG plugin and point it toward the local server
> you just ran(rather than the remote server it was used to looking for), but
> it should work.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/889#discussioncomment-2011258>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6H6EP4XGRZQJQ6S4O3UXC2BDANCNFSM5MN2A4DQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by apohl79** [comment]
As suggested, run AG server on the DAW machine you have on the road and run it in your studio. If you use version 1.2 beta, there is a tray icon for the plugins where you can reconnect all plugins to another server with a single click. So the switch would be easy.
